index.js

```diff
@@ -1,29 +1,57 @@
 /**
- * Add $search to `service.find` query. Be sure to text index your fields,
- * as it uses mongodb $text: https://docs.mongodb.com/manual/reference/operator/query/text/
+ * Add $search to `service.find` query. For text search queries on string content be sure to text index your fields,
+ * as it uses mongodb $text: https://docs.mongodb.com/manual/reference/operator/query/text/.
+ * For simple fuzzy match on a string field https://docs.mongodb.com/manual/reference/operator/query/regex/ is used.
  *
  * Usage:
  * ```
  * const fuzzySearch = require('feathers-mongodb-fuzzy-search')
  *
  * // ensure we have a text index
- * app.service('test').Model.createIndex({ title: 'text' })
+ * app.service('messages').Model.createIndex({ title: 'text content' })
```

Comment:
Again, create an index of type `text` for the field `title`.

Reply:
Yes you're right my bad